lib/fungus_toast_web/controllers/active_game_controller.ex

```diff
@@ -0,0 +1,12 @@
+defmodule FungusToastWeb.ActiveGameController do
+  use FungusToastWeb, :controller
+
+  alias FungusToast.Games
+
+  action_fallback FungusToastWeb.FallbackController
+
+  def show(conn, %{"id" => id}) do
+    active_game = Games.get_active_game!(id)
```

Comment:
Will there be a corresponding active game channel for this id?

Sorry about the questions, I should probably join y'all for a pairing session to get up to speed on this.